documents/design/fine_tuning_teb_parameters.md

```diff
@@ -0,0 +1,39 @@
+# Fine Tuning TEB Parameters
+
+*Issue #628*
+
+**Author:**
+- Amy Yao
+
+## The Problem
+
+The problem I am trying to solve is to fine tune the timed-elastic-bands control parameters for the
+robot.
+
+## Proposed Solution
+
+- I will first watch more videos and read more articles abut timed-elastic-bands control to make sure
+  that I understand how TEB works. I will learn what the appropriate values for each paramater
+  are and what will changing each of the variables do.
+- Then, I will try to use rviz to adjust, test, and tune the parameters before I make changes to the file.
```

Comment:
rviz is just the visualization tool. To tune the parameters you need to edit the `.yaml` file outside in an editor